src/MSBuild.Conversion.Facts/XamarinFacts.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MSBuild.Conversion.Facts
+{
+    public static class XamarinFacts
+    {
+        public const string Net6XamarinAndroid = "net6.0-android";
+        public const string Net6XamariniOS = "net6.0-ios";
+
+        public static ImmutableArray<string> UnnecessaryXamProperties => ImmutableArray.Create(
+            // The following are unecessary in .NET MAUI
+
+            "Configuration",
+            "Platform",
+            "ProductVersion",
+            "SchemaVersion",
+            "ProjectGuid",
+            "ProjectTypeGuids",
+            "TemplateGuid",
+            "AppDesignerFolder",
+            "RootNamespace",
+            "AssemblyName",
+            "FileAlignment",
+            "Deterministic",
+            "TargetFrameworkVersion",
+            "DebugSymbols",
+            "DebugType",
+            "Optimize",
+            "OutputPath",
+            "DefineConstants",
+            "ErrorReport",
+            "WarningLevel",
+            "GenerateAssemblyInfo",
+            "GenerateSerializationAssemblies"
+
+            //Droid
+            /***
+           "AndroidApplication",
```

Comment:
Why commented out?

Reply:
ooH WOOPS totally forgot to delete this! It wasn't meant to be left in, I was testing stuff and totally forgot 😅